src/main/resources/db/changelog/changes/07-insert-users-in-user-table.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-users-in-users-table
+      author: fetis_off
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: "admin@gmail.com"
+              - column:
+                  name: password
+                  value: "1234"
+              - column:
+                  name: first_name
+                  value: "Audi"
```

Comment:
minor comment 
This project is for you portfolio so it's better to provide ordinal first and last names